qiskit/transpiler/passes/synthesis/unitary_synthesis.py

```diff
@@ -350,8 +349,6 @@ def __init__(
                 directionality of the coupling_map will be taken into
                 account if ``pulse_optimize`` is ``True``/``None`` and ``natural_direction``
                 is ``True``/``None``.
-            backend_props (BackendProperties): Properties of a backend to
-                synthesize for (e.g. gate fidelities).
```

Comment:
Ah, I see, the `BackendProperties` should be removed by the "prequel" PR #13706, correct?